### DISCUSSION TITLE
Still turning to early

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5605] [Q&A]
Hi dev team,

I think you are still in developing the flight plan manager .... still turning much to early and still an Asobo bug :-p ...

.. in the pic NAV mode allready disengaged !!! 

![grafik](https://user-images.githubusercontent.com/76531893/128863032-2fb10975-684f-4ff5-9bba-80d9e1749409.png)

Greets Klaus

**Comment by tracernz** [comment]
The new flight planning and guidance system will be of great interest to you. ;)
